encrypted-media/scripts/playback-persistent-license-events.js

```diff
@@ -0,0 +1,168 @@
+function runTest(config, testname) {
+
+    var testname = config.keysystem + ', successful playback, persistent-license, '
+                                    + /video\/([^;]*)/.exec( config.videoType )[ 1 ]
+                                    + ', set src before setMediaKeys, check events';
+
+    var configuration = {   initDataTypes: [ config.initDataType ],
```

Comment:
It can make side-by-side diffing more difficult, and GitHub doesn't handle long lines well. This was a general comment and not specific to this line. It looks like we're using four-space tabs, which has a similar effect. Do WPT have a preferred style?